image-manipulation.py

```diff
@@ -0,0 +1,11 @@
+from PIL import ImageEnhance
+import os
+size_256 = (256,256)
+for f in os.listdir('.'):
+    if f.endswith('.jpg'):
+        i = Image.open(f).convert('RGB')
+        fn, fext = os.path.splitext(f)
+        enhancer = ImageEnhance.Brightness(i)
+        i = enhancer.enhance(1.5)
+        img = i.resize(size_256, Image.ANTIALIAS)
+        img.save('Clean_darth vader/{}_clean{}'.format(fn, fext))
```

Comment:
Code should always have a newline at end :-) Just a good practice
Why?
If you press Ctrl+End button in an editor, you expect it to end in last line not on last letter in last line